util/server-watch.sh

```diff
@@ -1,9 +1,35 @@
 #!/bin/bash
 set -e
 
+usage() {
+  USAGE="Usage: server-watch.sh [-d]
+    -d : Start a debugging session with Delve"
+  echo "${USAGE}"
+}
+
+while getopts ':dh' flag; do
+  case "${flag}" in
+    d) DEBUG='true' ;;
+    h|*) usage && exit 0;;
+  esac
+done
+UTIL_DIR=$(dirname $0)
+source "${UTIL_DIR}/logging.sh"
+
 make go_build_dev
-./web &
-SERVER_PID=$!
+if [[ ${DEBUG} != "true" ]];
+then
+  ./web &
+  SERVER_PID=$!
+else
+  if [[ $(which dlv) == "" ]]; then \
+    go install github.com/go-delve/delve/cmd/dlv@latest
+  fi
+  info "Starting debugger on port 12345"
+  dlv debug github.com/web-platform-tests/wpt.fyi/webapp/web --headless --listen=:12345 --output /tmp/web.debug
+  exit 0
```

Comment:
Currently, debug mode will not watch for changes and rebuild. You may want to modify your info print to indicate that.

Reply:
Done, although there is no console message that rebuilding on changes is enabled as far as I can see.

I also made the `wptd_exec make inotifywait` line in web_server.sh conditional on `${DEBUG} != "true"`.